src/components/UserManagement/UserOrderHistory/UserOrderHistory.module.scss

```diff
@@ -1,9 +1,14 @@
+@import "styles/mixins";
+
 .UserOrderHistory {
   padding: 62px 100px 162px;
 
-  @media (max-width: 970px) {
-    padding: calc(var(--vertical-spacing-normal) * 2)
-      calc(var(--vertical-spacing-normal) * 2) 50px;
+  //@media (max-width: 970px) {
+  //  padding: calc(var(--vertical-spacing-normal) * 2)
+  //    calc(var(--vertical-spacing-normal) * 2) 50px;
+  //}
+  @include mobile-media {
+    padding: 0;
```

Comment:
Not sure if this is related, but let's add a bit of margin to the Load More button :D 
https://gyazo.com/7ecd3f29ea7d7bbb2d623fa3ba7036ab